roboteam_ai/src/utilities/Constants.h

```diff
@@ -49,12 +49,13 @@ class Constants {
     static int TICK_RATE()                      { return 60; };
     static double CLOSE_TO_BORDER_DISTANCE()    { return 1.2 * ROBOT_RADIUS(); };
     static int GAME_ANALYSIS_TICK_RATE()        { return 30; };
+    static double DISTANCE_TO_KICKER()          { return 0.06; };
```

Comment:
This is almost exactly the same as CentreToFront() (which is ~0.059) which is the exact distance to the front of the robot. I suggest we don't make duplicate constants for the same concept (so either change this one or use the other one that does the proper measurement).

Reply:
Essentially this is just a duplicate.